### DISCUSSION TITLE
Show name of the first plugin inserted.

### DISCUSSION THREAD

**Discussion by den0rk** [apohl79/audiogridder#388] [Ideas]
AA users are beginning to discover that using audiogridder as a plugin host in local mode, that is to say, server on the same pc, helps to run many more AA plugins than on the DAW, since the only efficient DAW is Reaper.

I think it would be a good idea if the plugin name in the DAW shows the name of the first plugin inserted into AudioGridder, obviously this should be optional in the settings.

**Comment by CaBuffalo** [comment]
Thanks for the good idea.

On Fri, Feb 19, 2021 at 2:53 AM den0rk <notifications@github.com> wrote:

> AA users are beginning to discover that using audiogridder as a plugin
> host in local mode, that is to say, server on the same pc, helps to run
> many more AA plugins than on the DAW, since the only efficient DAW is
> Reaper.
>
> I think it would be a good idea if the plugin name in the DAW shows the
> name of the first plugin inserted into AudioGridder, obviously this should
> be optional in the settings.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/388>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6ERN6LLG2JKLVUCUEDS7Y7JJANCNFSM4X4E3VAA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by kastorjunior** [reply]
It's possible in Reaper but I don't know for the other DAW's.

**Comment by apohl79** [comment]
I'm not sure that this is possible. VST and AU seem not to be designed to allow for plugins to propagate an updated name to the host. It might be possible, but JUCE does not support this, so nothing that can be added easily right now.

**Comment by den0rk** [comment]
Oh :(

In the case of Reaper, you can do the manual work, replace the name and save it as an FX chain, so then with just 1 click it will load AGridder with the plugin already inserted and the name established.
